includes/DependencyManagement/Providers/AdminDashboardServiceProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace WeDevs\Dokan\DependencyManagement\Providers;
+
+use WeDevs\Dokan\Admin\Dashboard\Dashboard;
+use WeDevs\Dokan\Admin\Dashboard\Pages\Status;
+use WeDevs\Dokan\DependencyManagement\BaseServiceProvider;
+
+class AdminDashboardServiceProvider extends BaseServiceProvider {
+    /**
```

Comment:
Pls check the [similar implementation](https://github.com/getdokan/dokan/blob/a169e59f4af7870e4e433bae212a81c4757eec2c/includes/DependencyManagement/Providers/AnalyticsServiceProvider.php#L41)